app/routes/index.js

```diff
@@ -215,6 +215,15 @@ export default class IndexRoute extends Route {
           op   : 'ge',
           val  : params.start_date
         }
+      },
+      {
+        name : 'speakers-call',
+        op   : 'has',
+        val  : {
+          name : 'announcement',
+          op   : 'ne',
+          val  : ''
```

Comment:
If speaker call is being disabled, announcement should be set to null, not empty string

Reply:
it's giving error `field may not be null`.

Reply:
Make it nullable in server please

Reply:
`announcement = db.Column(db.Text, nullable=True)` do i have to migrate also after making it nullable.

Reply:
Any change in schema means migration